### DISCUSSION TITLE
Remove custom-bundle mojo?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-tycho/tycho#929] [Polls]
I'd like to remove the `custom-bundle` I can't find any use-case for this except the mojo itself and the mojo.

Given that we now support mixed reactor builds it might be obsolete anyways?

**Comment by akurtakov** [comment]
See https://github.com/eclipse-jdt/eclipse.jdt.core/blob/master/org.eclipse.jdt.core/pom.xml#L74 for usage. I know that things can always be changed but there are probably other usages like that too.

**Reply by laeubi** [reply]
Can you explain what it does? I can't find any documentation about the usage/use cases anywhere, e.g. what is a "custom" bundle in contrast to a regular one?

**Reply by akurtakov** [reply]
It creates multiple artifacts with different content in the same module. See the various execution sections of the plugin config. A custom bundle is literally one that you manually assemble content of it.
